contrib/openbsm/bin/bsmconv/bsmconv.c

```diff
@@ -0,0 +1,58 @@
+#include <stdio.h>
+#include <nv.h>
+#include <unistd.h> /* getopt(3) */
+
+#include "linau.h"
+
+#include "pjdlog.h"
+
+int
+main(int argc, char *argv[]) {
+	linau_event *event;
+	linau_record *record;
+	FILE * fp;
+	size_t counter;
+	int debuglevel;
```

Comment:
Style: wrong variables order. See style(9). There should be no space between an asterisk and a variable name.